<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diferenciacion</class>
 <widget class="QMainWindow" name="diferenciacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>diferenciacion</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>xi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>X(i+1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>X(i+2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>X(i+3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>X(i+4)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>X(i-1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>X(i-2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>X(i-3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>X(i-4)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_5"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_14"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_16"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_7"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_9"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_12"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_18"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>f(xi)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>FX(i+1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>FX(i+2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>FX(i+3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>FX(i+4)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>FX(i-1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>FX(i-2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>FX(i-3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>FX(i-4)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_6"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_15"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_17"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_8"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_10"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_13"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_19"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>h=</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_11"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>PUSH pARa MeToDo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Borrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
